pkg/upload/react-native-android.go

```diff
@@ -35,15 +35,19 @@ func ProcessReactNativeAndroid(apiKey string, appManifestPath string, bundlePath
 	for _, path := range paths {
 
 		buildDirPath := filepath.Join(path, "android", "app", "build")
+		rootDirPath := path
 		if !utils.FileExists(buildDirPath) {
 			buildDirPath = filepath.Join(path, "app", "build")
+			rootDirPath = filepath.Join(path, "..")
 			if !utils.FileExists(buildDirPath) {
 				if bundlePath == "" || sourceMapPath == "" {
 					return fmt.Errorf("unable to find bundle files or source maps in within " + path)
 				}
 			}
 		}
 
+		projectRoot = rootDirPath
```

Comment:
We still only want to to do this if it's not specified in the option:

```suggestion
		if projectRoot == "" {
			projectRoot = rootDirPath
		}
```